docs/testenv.md

```diff
@@ -0,0 +1,75 @@
+# Running the Joystream ecosystem locally
+
+The Pioneer 2 is targeted for the Olympia release and it is compatible with the runtime from the `olympia` branch.
+
+```
+https://github.com/thesan/joystream/joystream/olympia
+```
+
+However, the current version of Pioneer 2 is ahead of the olympia branch in terms of the query-node support. While the query-node related features are under development, we track the most recent changes at:
+
+```
+https://github.com/thesan/joystream/tree/debug-pioneer-elections
+```
+In order to use the compatible version of the Joystream ecosystem
+
+## Prepare the dev environment
+
+### 1. Install tools
+
+This guide assumes that [docker](https://www.docker.com/) is installed.
+
+### 2. Checkout integration branch
+
+```shell
+git origin add thesan https://github.com/thesan/joystream/tree/debug-pioneer-elections
+git fetch
+git checkout debug-pioneer-elections
+```
+
+### 3. Install dependencies
+
+Run `yarn install` inside the main repo. The full setup is not need (especially if you'd like to avoid using Volta).
+
```

Comment:
`yarn run build:packages` starts with installing the dependencies so I think this step could be skipped